.github/workflows/version-or-publish.yml

```
@@ -0,0 +1,30 @@
+name: Version or Publish
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  changesets:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Repo
+        uses: actions/checkout@master
+
+      - name: Setup Node.js 12.x
+        uses: actions/setup-node@master
+        with:
+          node-version: 12.x
+
+      - name: Install Dependencies
+        run: yarn
+
+      - name: Create Release Pull Request or Publish
+        id: changesets
+        uses: changesets/action@master
```

Shouldn't we point to the local built action?

```suggestion
        uses: .
```

This requires of course to build the code first.

---

Probably - yes. I wasn't sure how to make it work though, are you sure that this will work?

---

Yes it should work (see example here https://docs.github.com/en/actions/creating-actions/creating-a-javascript-action#example-using-a-private-action)